scripts/getTopic.js

```diff
@@ -0,0 +1,20 @@
+const { validateNotEmpty } = require('./library/validation');
+
+/**
+ * Returns topic
+ * @param  {object} azureServiceBus - Service Bus Instance
+ * @param  {string} topic - Topic to use
+ * @return {Promise}
+ */
+function getTopic(azureServiceBus, topic) {
+    return new Promise((resolve, reject) => {
+        validateNotEmpty([azureServiceBus, topic], reject);
+
+        azureServiceBus.getTopic(topic, (error, gettopicresult) => {
+            if (error) return reject(error);
+            return resolve({gettopicresult});
+        });
+    });
+}
+
+module.exports.getTopic = getTopic;
```

Comment:
I'm not entirely sure but I think github is showing me a line ending that isn't uniform.

Reply:
new line added after line 20